sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -294,6 +312,37 @@ should take ownership of the passed native handle. The default behavior is to
 transfer the ownership to the SYCL runtime. See section 4.4 for details. If
 the behavior is "transfer" then the runtime is going to destroy the input
 Level-Zero kernel</td>
+</tr><tr>
+<td>
+
+``` C++
+make_buffer(
+    const backend_input_t<backend::ext_oneapi_level_zero,
+                          buffer<T, Dimensions, AllocatorT>> &,
+    const context &Context)
+```
+</td>
+<td>This API is available starting with revision 2 of this specification.
+
+Construct a SYCL buffer instance from a pointer to a Level Zero memory buffer. The pointer must be the value returned from a previous call to <code>zeMemAllocShared()</code>, <code>zeMemAllocDevice()</code>, or <code>zeMemAllocHost()</code>. If pointer is the value returned from <code>zeMemAllocDevice()</code> or <code>zeMemAllocShared()</code> then SYCL context <code>Context</code> must be associated with a single device, matching the device used at the prior allocation.

```

Comment:
The  `zeMemAllocShared` API does not require a device to be passed.  Is it allowed to call `make_buffer()` in this case?  If so, what device (if any) should `Context` contain?

Reply:
I think we should just say that if an USM allocation is bound to a device, then that device must be present in the context being passed. Then, we should be able to "pick up" any USM allocation and organize a valid SYCL buffer with that (it may mean creating more allocation for other devices in the context).

Reply:
And it is a temporary requirement (orthogonal to other standing ones) that only single-device contexts are supported currently.